cli_tools/gce_ovf_import/ovf_utils/ovf_utils.go

```diff
@@ -36,6 +36,36 @@ const (
 	usbController          uint16 = 23
 )
 
+//TODO: add Windows 7 and 10 once BYOL is supported
+//Full list: https://www.vmware.com/support/developer/vc-sdk/visdk41pubs/ApiReference/vim.vm.GuestOsDescriptor.GuestOsIdentifier.html
+
+// Mapping OVF osType attribute to importer OS ID
+var ovfOSTypeToOSID = map[string]string{
```

Comment:
Just fyi I'm pretty sure these fields in vmware mean close to nothing, so users could be setting incorrect values with really no negative side effects. The issue it could cause with import is that the import will fail later on.

Reply:
These values are exported in OVF by VMWare. Users will probably not set them themselves. 

As you pointed out, having these values set incorrectly would fail the import during translate. Not sure we can do much about it, but I doubt this will be an issue, except if users are modifying OVF descriptor themselves.